src/elements/content-sharing/messages.js

```diff
@@ -53,6 +53,12 @@ const messages = defineMessages({
             'Message that appears when collaborators cannot be added to the shared link in the ContentSharing Element.',
         id: 'be.contentSharing.sendInvitesError',
     },
+    sendInvitesExistingCollabError: {
+        defaultMessage: 'User is already a collaborator.',
```

Comment:
Just wondering, does the API response let us know which user is already a collaborator?